oracle/Dockerfile

```diff
@@ -19,8 +19,8 @@ RUN \
   curl -L -o sbt-$SBT_VERSION.rpm https://bintray.com/sbt/rpm/download_file?file_path=sbt-$SBT_VERSION.rpm && \
   rpm -i sbt-$SBT_VERSION.rpm && \
   rm sbt-$SBT_VERSION.rpm && \
-  yum update && \
-  yum install sbt
+  yum -y update && \
```

Comment:
Nice! Did that ever work before?

Reply:
nope, must not have been tested as it prompted if it was ok to download